### PR TITLE
PLANNER-435 - Workbench: Data modeler's planning solution should implement Solution interface

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

File: optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml [hunk 1]
```diff
@@ -38,6 +38,10 @@
       <artifactId>gwtbootstrap3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.gwtbootstrap3</groupId>
```
Comment (manstis) [inline optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml:41]: If you're adding use of GWT-BS3's `Select` you also need include an additional `gwt.xml` module.

i.e. `<inherits name="org.gwtbootstrap3.extras.select.Select"/>`

See https://gwtbootstrap3.github.io/gwtbootstrap3-demo/#select

I couldn't see that in this commit.

Comment (wmedvede) [inline optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml:41]: I can add it if needed, but I've looked how Alexandre was added it to the
datamodeller during BS3 migration and the <inherits
name="org.gwtbootstrap3.extras.select.Select"/>
wasn't added there.

2015-10-01 14:34 GMT+02:00 Michael Anstis notifications@github.com:

**Comment by manstis** [comment]
OK, I'll leave it to your discretion (IDK if porcelli could guarantee the "extra" module was already on the inheritance chain). I know Roger had similar issues on Dash Builder.. an "extra" module was omitted and kie-(drools)-wb webapp's failed to compile.

**Comment by wmedvede** [comment]
No problem, I can add it.

2015-10-01 15:43 GMT+02:00 Michael Anstis notifications@github.com:

**Pull request merged**